pkg/kn/core/root_test.go

```diff
@@ -40,6 +41,51 @@ func TestNewDefaultKnCommand(t *testing.T) {
 	})
 }
 
+func TestNewDefKnCmdWithArgsforCmdValiditySuccess(t *testing.T) {
+
+	pluginHandler := plugin.NewDefaultPluginHandler(plugin.ValidPluginFilenamePrefixes,
+		"/random/plugins", false)
+	args := []string{"kn", "service", "update", "--tag=0.13"}
+	rootCmd := NewDefaultKnCommandWithArgs(NewKnCommand(), pluginHandler, args, os.Stdin, os.Stdout, os.Stderr)
+	checkRootCmd(t, rootCmd)
+}
+
+func TestNewDefKnCmdWithArgsforPrimaryCmdValidityErr(t *testing.T) {
+
+	if os.Getenv("EXIT") == "1" {
+		pluginHandler := plugin.NewDefaultPluginHandler(plugin.ValidPluginFilenamePrefixes,
+			"/random/plugins", false)
+		args := []string{"kn", "srevice", "update", "--tag=0.13"}
```

Comment:
Did you mean to have a typo in “service” here as a bad verb?

Reply:
This is to test if primary command "service"  is mispelt